lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -412,6 +412,13 @@ LogicalResult CreateSiFiveMetadataPass::emitSitestBlackboxMetadata() {
 
   createOutput(testModules, testFilename);
   createOutput(dutModules, dutFilename);
+
+  // Clean up all ScalaClassAnnotations, which are no longer needed.
+  for (auto op : circuitOp.getOps<FModuleLike>()) {
+    AnnotationSet annos(op);
+    annos.removeAnnotations(op, scalaClassAnnoClass);
+  }
```

Comment:
There's a slightly tighter way the PR can express this:
```suggestion
  for (auto op : circuitOp.getOps<FModuleLike>())
    AnnotationSet::removeAnnotations(op, scalaClassAnnoClass);
```

Reply:
oh nice, i missed that helper, will use it.